src/games/strategy/triplea/ai/proAI/ProAI.java

```diff
@@ -223,25 +209,29 @@ protected void purchase(final boolean purchaseForBid, int PUsToSpend, final IPur
         final String stepName = step.getName();
         ProLogger.info("Simulating phase: " + stepName);
         if (stepName.endsWith("NonCombatMove")) {
+          ProData.initialize(this, dataCopy, playerCopy);
           final Map<Territory, ProTerritory> factoryMoveMap =
-              nonCombatMoveAI.doNonCombatMove(null, null, moveDel, dataCopy, playerCopy, true);
+              nonCombatMoveAI.doNonCombatMove(null, null, moveDel, true);
```

Comment:
Agree. Creating new 'simulateNonCombatMove' method and moving the boolean into the ProData class.